lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -0,0 +1,182 @@
+//===- FIRAnnotations.cpp - .fir to FIRRTL dialect parser -----------------===//
```

Comment:
Plz update comment line, this is about annotations not the parser as a whole

Reply:
Fixed in rebase.